concrete/single_pages/dashboard/system/basics/multilingual/view.php

```diff
@@ -42,7 +42,7 @@
 
 if (isset($mlLink)) {
     ?>
-    <div class="alert alert-info small">
+    <div class="ccm-dashboard-content-indented alert alert-info small">
```

Comment:
I don't think we need this new class. Let's just use ml-3 or ml-2 if we need something like that.

Reply:
**Same comment as above:**

form-group elements all get a custom padding-left of 20px. Using BS4 ml-x classes we have either 16px or 24px so it doesn't align with the other stuff on the page, hence the custom class.
How do you want me to deal with that?

One possibility is to use a form-group wrapper for everything even if not dealing with a form. Would that be acceptable?